pytorch_lightning/utilities/cloud_io.py

```diff
@@ -19,7 +19,8 @@
 
 import fsspec
 import torch
-from fsspec.implementations.local import AbstractFileSystem, LocalFileSystem
+from fsspec.core import url_to_fs
```

Comment:
is there any specific version after which this function is available in `fsspec`? If so, we might need to update the minimum requirements: https://github.com/PyTorchLightning/pytorch-lightning/blob/ec1379da2c9388b144adcb4b4cc751839f085f65/requirements.txt#L7

else it's fine.

Reply:
it's been available for at least a year, but hadn't been publicly documented until this: https://github.com/fsspec/filesystem_spec/issues/752

Reply:
that's a great point! It's available after [2021.09.0](https://github.com/fsspec/filesystem_spec/releases/tag/2021.09.0)
I'll update it.